### ISSUE TITLE
Incorrect syntax / use for heading in sidebar

### ISSUE THREAD

**Issue by ChrisXKeroack** [intel/openfl#858]
**Describe the bug**
Sidebar uses a heading without declaring a heading level

**To Reproduce**
1. https://openfl.readthedocs.io/en/latest/index.html
2. Examine the code for the 'CONTENTS:' text

notice: Sidebar declares a heading for ‘contents’ via role=heading but doesn’t set a heading level

**Expected behavior**
add aria-heading="2"

**Desktop:**
This issue is not OS or browser specific, but for tracking purposes, testing was done on Win11, in Edge Version 114.0.1823.58 (Official build) (64-bit).

**Smartphone:**
This issue is not device, OS, or browser specific.

**Additional context**
This is an accessibility issue, please annotate this bug with the 'accessibility' label.

**Comment by fstrr** [comment]
This is an issue with the [Read The Docs platform](https://github.com/readthedocs/readthedocs.org).

Note: the [ARIA 1.1](https://www.w3.org/TR/wai-aria-1.1/#heading) specification allowed the use of `role=heading` without an accompanying `aria-level` attribute, and would default to a heading level of 2 if `aria-level` was missing. The [ARIA 1.2](https://www.w3.org/TR/wai-aria-1.2/#heading) specification forbids that and requires an `aria-level` attribute.

**Comment by rahulga1** [comment]
As this issue with Read the Docs platform, it is outside of openFL repo, hence closing it.
